go/type.go

```diff
@@ -310,6 +310,10 @@ func (self *Compiler) compile_dict(n *VimNode) string {
 	}
 }
 
+func (self *Compiler) compile_parenexpr(n *VimNode) string {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
don't use underscores in Go names; method compile_parenexpr should be compileParenexpr